lib/vjs-hls.js

```diff
@@ -30,7 +31,49 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
         }
 
         function _onError(event, data) {
-            console.error('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details);
+            var error = {
+              message: ('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details),
+            };
+            console.error(error.message);
+
+            // increment/set error count
+            errorCounts[data.type] ? errorCounts[data.type] += 1 : errorCounts[data.type] = 1;
```

Comment:
I had forgotten to change the function declaration with the additional arguments.

I was not able to to access `_errorCounts` or `tech` directly from within the `_onError` function even though it is defined within the same scope as those variables.  that's why I use another anonymous function above to pass them to `_onError`